crmsh/bootstrap.py

```diff
@@ -508,6 +509,17 @@ def _update_configuration(self):
         utils.sysconfig_set(SYSCONFIG_SBD, **sbd_config_dict)
         csync2_update(SYSCONFIG_SBD)
 
+    def _determine_watchdog_timeout(self):
+        """
+        Determine value of stonith-watchdog-timeout
+        """
+        conf = utils.parse_sysconfig(SYSCONFIG_SBD)
+        res = con.get("SBD_WATCHDOG_TIMEOUT")
+        if res:
+            self._watchdog_timeout = -1
+        elif "390" in os.uname()[4]:
```

Comment:
`os.uname().machine` for better readability

Reply:
Changed, thanks!